src/geo.c

```diff
@@ -176,8 +176,8 @@ int extractDistanceOrReply(client *c, robj **argv,
 int extractBoxOrReply(client *c, robj **argv, double *conversion,
                          double *height, double *width) {
     double h, w;
-    if ((getDoubleFromObjectOrReply(c, argv[0], &h, "need numeric height") != C_OK) ||
-        (getDoubleFromObjectOrReply(c, argv[1], &w, "need numeric width") != C_OK)) {
+    if ((getDoubleFromObjectOrReply(c, argv[0], &h, "need numeric width") != C_OK) ||
+        (getDoubleFromObjectOrReply(c, argv[1], &w, "need numeric height") != C_OK)) {
```

Comment:
looks like you only fixed the error message, but not the actual parameters
```suggestion
    if ((getDoubleFromObjectOrReply(c, argv[0], &w, "need numeric width") != C_OK) ||
        (getDoubleFromObjectOrReply(c, argv[1], &h, "need numeric height") != C_OK)) {
```